iphone/Classes/TiLogServer.m

```diff
@@ -0,0 +1,347 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+/**
+ * Important: Do NOT call NSLog() from this file. The app will go into an
+ * infinite loop of death. Use forcedNSLog() instead.
+ */
+
+#ifndef DISABLE_TI_LOG_SERVER
+
+#import "TiLogServer.h"
+#import "TiBase.h"
+#include <sys/socket.h>
+#include <netinet/in.h>
+
+/**
+ * These constants are available to allow xcodebuild to override them.
+ */
+#ifndef TI_LOG_SERVER_PORT
+# define TI_LOG_SERVER_PORT 10571
```

Comment:
But there are instances elsewhere in the code that is formatted this way. Usually I put a few more spaces between the # and "define", but stuck with 1 for consistency. This is a good and very old practice.